cmake/FindLIBHWLOC.cmake

```diff
@@ -55,7 +55,7 @@ if(WINDOWS)
 endif()
 
 if(LIBHWLOC_LIBRARY)
-    message(STATUS "  Found libhwloc using find_library()")
+    message(STATUS "  Found libhwloc: ${LIBHWLOC_LIBRARY}")
 
```

Comment:
perhaps msg down below now should be updated a little
`libhwloc NOT found (set CMAKE_PREFIX_PATH to point the location or disable with -DUMF_DISABLE_HWLOC=ON)`

saying that it falls back to fetching...?